geopy/geocoders/herev7.py

```diff
@@ -0,0 +1,286 @@
+import collections.abc
+import warnings
+from functools import partial
+from urllib.parse import urlencode
+
+from geopy.exc import (
+    ConfigurationError,
+    GeocoderAuthenticationFailure,
+    GeocoderInsufficientPrivileges,
+    GeocoderQuotaExceeded,
+    GeocoderServiceError,
+    GeocoderUnavailable,
+)
+from geopy.geocoders.base import DEFAULT_SENTINEL, Geocoder
+from geopy.location import Location
+from geopy.util import join_filter, logger
+
+__all__ = ("HereV7", )
+
+
+class HereV7(Geocoder):
+    """Geocoder using the HERE Geocoder API.
```

Comment:
Could you please describe the difference from the `Here` class, so it would be clear which one to use? (An also in `Here` as well). Example:
- https://github.com/geopy/geopy/blob/24b9226e1539a96ce379fad0b457fcfe37b3d6be/geopy/geocoders/baidu.py#L24-L26
- https://github.com/geopy/geopy/blob/24b9226e1539a96ce379fad0b457fcfe37b3d6be/geopy/geocoders/baidu.py#L276

Reply:
Fixed in 445fbac